src/test/java/Homework17.java

```diff
@@ -35,8 +33,28 @@ public void enterPassword(String password) {
         passwordField.click();
         passwordField.sendKeys(password);
         }
-        public void clickSubmit() {
-        WebElement submitButton = driver.findElement(By.cssSelector("button[type='submit']"));
-        submitButton.click();
+        public void allSongs() throws InterruptedException {
+        WebElement allSongs = driver.findElement(By.cssSelector("#sidebar > section.music > ul > li:nth-child(3) > a"));
+        allSongs.click();
+        Thread.sleep(2000);
+        }
+        public void songName() throws InterruptedException {
```

Comment:
**I recommend to use in the methods naming:** verb, long names and follow code convention

_For example_   `public void enterSongName()`